### DISCUSSION TITLE
How to disable admin apps

### DISCUSSION THREAD

**Discussion by cooltoo37** [3kh0/ext-remover#575] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is their anyway that you could disable a admin app using exploits and the admin app is called Cisco Umbrella Chromebook client (App)
 I have version 102.0.5005.197 
Thanks

**Comment by Aka-but-nice** [comment, marked as answer]
Hmm, not so sure as, the exploit Ingot nor LTBEEF has the power to disable apps, DM for a exploit.

**Reply by cooltoo37** [reply]
dm you on discord or waht
